pkg/controller/servicemesh/webhooks/validation/member.go

```diff
@@ -53,7 +53,8 @@ func (v *MemberValidator) Handle(ctx context.Context, req admission.Request) adm
 		return admission.Errored(http.StatusBadRequest, fmt.Errorf("ServiceMeshMember must be named %q", common.MemberName))
 	}
 
-	if smm.Namespace == common.GetOperatorNamespace() {
+	// TODO: Need switch for external/split control plane detection (Dataplane is in a different clustern than the operator)
+	if !common.Config.OLM.SplitModeEnabled && smm.Namespace == common.GetOperatorNamespace() {
```

Comment:
Ah, this is where you can't/shouldn't get the SMCP and replace SplitModeEnabled with !SMCP.ControlPlaneMode.

Reply:
Yes, because this webhook is responsible for validating the SMCP